x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/job_map_title.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const JobMapTitle: FC<{ analyticsId?: string; modelId?: string }> = ({
+  analyticsId,
+  modelId,
+}) => (
+  <EuiTitle size="xs">
+    <span>
+      {analyticsId
+        ? i18n.translate('xpack.ml.dataframe.analyticsMap.analyticsIdTitle', {
```

Comment:
According to Kiabna guidelines, `FormattedMessage` is preferred over `i18n` in react components

Reply:
Updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9